scripts/run_node.sh

```diff
@@ -266,8 +266,8 @@ then
         NC='\033[0m'
 
         >&2 echo -e "${RED}"
-        >&2 echo "Session keys are set for this stash account, but it seems like you do not have access to them."
-        >&2 echo "You might want to generate new keys and set them for your stash account."
+        >&2 echo "Session keys are set for this controller account, but it seems like you do not have access to them."
+        >&2 echo "You might want to generate new keys and set them for your controller account."
```

Comment:
In general, regenerating keys should not be encouraged, because the new keys will be pending for ~2 sessions, which means downtime. Instead we should recommend migrating the old keys (they are in the keystore folder), and only if that's impossible, then regenerate.